src/script/keyorigin.h

```diff
@@ -18,6 +18,21 @@ struct KeyOriginInfo
         return std::equal(std::begin(a.fingerprint), std::end(a.fingerprint), std::begin(b.fingerprint)) && a.path == b.path;
     }
 
+    friend bool operator<(const KeyOriginInfo& a, const KeyOriginInfo& b)
+    {
+        for (int i = 0; i < 4; ++i) {
```

Comment:
You can probably write this more readably as:

```
if (std::lexicographic_compare(std::begin(a.fingerprint), std::end(a.fingerprint), std::begin(b.fingerprint), std::end(b.fingerprint)) return true;
if (std::lexicographic_compare(std::begin(b.fingerprint), std::end(b.fingerprint), std::begin(a.fingerprint), std::end(a.fingerprint)) return false;
```

etc., which also avoids the magic 4 number.

Reply:
Done